composer.json

```diff
@@ -36,7 +36,7 @@
 		"php": "^7.3||^7.4",
 		"typo3/cms-core": "^10.4",
 		"typo3/cms-extbase": "^10.4",
-		"lochmueller/calendarize": "^10.0||^11.0",
+		"lochmueller/calendarize": "^10.0||^11.0|^12",
```

Comment:
Note: there is a missing "|" and please add ".0" after the "12".

Reply:
Oops, is fixed.

Please note I did not perform any check or test yet if the calendarize_news is already compatible with calenderize v12 yet. Hence the "Draft" status. But we will be testing this in the next days.

I already created the PR here, in case anyone else is working on this or testing this and can comment and also as reminder for myself to not loose track of my contributions.

Thanks for your great work and speedy merging @lochmueller.